staging/src/k8s.io/kubectl/pkg/cmd/create/create_rolebinding_test.go

```diff
@@ -22,7 +22,7 @@ import (
 
 	rbac "k8s.io/api/rbac/v1"
 	apiequality "k8s.io/apimachinery/pkg/api/equality"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
don't need a `v1` here :)

Reply:
thanks, it's still the go linter (maybe gofmt)